server/game/cycle/stichGranter.js

```diff
@@ -2,98 +2,54 @@ import {GameMode} from '../../../shared/game/gameMode';
 
 const StichGranter = {
     determineWinner: function (mode, trumpfColor, playedCards, players) {
-        let winnerIndex = 0;
-        let winnerCard;
+        const firstPlayedColor = playedCards[0].color;
+        const isModeTrumpf = mode === GameMode.TRUMPF;
+        const trumpfAndFirstPlayedColorCards = playedCards.map(addIndexAndAdjustTrumpfValues)
```

Comment:
maybe we can change the name of trumpfAndFirstPlayedColorCards ? wouldn't it be trumpfOrFirstPlayedColorCards? Or something like countingCards? If you get what I mean :)